pkg/config/types.go

```diff
@@ -72,6 +74,36 @@ func parseIncludeExcludeNamespacesFromNamespacesConfig(in string) (namespacesCon
 	return namespacesConfigObject, err
 }
 
+type metricExposureConfig struct {
+	Enabled                 *bool     `json:"enabled,omitempty"`
+	DisabledLabelDimensions []string  `json:"disabledLabelDimensions,omitempty"`
+	BucketBoundaries        []float64 `json:"bucketBoundaries,omitempty"`
```

Comment:
Do we need to expose these fields?

Reply:
Hi @realshuting , thanks for your comment, unfortunately those fields must be exposed because the `metricExposureConfig` struct is being unmarshalled from json in the `parseMetricExposureConfig` function, therefore the fields will only be populated from the json content if the fields are exposed. 

This the same case from the existing `namespacesConfig` struct, which is also exposing the `IncludeNamespaces` and `ExcludeNamespaces` for the exact same reason.

Reply:
Thanks @rodrigorfk !